core-bundle/contao/templates/twig/content_element/text.html.twig

```diff
@@ -23,7 +23,7 @@
     {% block text %}
         {% set text_attributes = attrs(text_attributes|default).addClass('rte') %}
         <div{% block text_attributes %}{{ text_attributes }}{% endblock %}>
-            {{ text|insert_tag|extract_styles_for_csp|raw }}
+            {{ text|extract_styles_for_csp|insert_tag|raw }}
```

Comment:
I think it should be this way around as insert tags  (e.g. {{insert_content::*}}) would handle CSP on their own.